src/Microbenchmarks.jl

```diff
@@ -0,0 +1,75 @@
+module Microbenchmarks
+
+using DataFrames, CSV, StatsBase
+
+function read_bench(benchfile::String)
+    # Load benchmark data from file
+    benchmarks = CSV.read(benchfile, DataFrame; header=["language", "benchmark", "time"])
+
+    # Capitalize and decorate language names from datafile
+    dict = Dict("c"=>"C", "julia"=>"Julia", "lua"=>"LuaJIT", "fortran"=>"Fortran", "java"=>"Java",
+                "javascript"=>"JavaScript", "matlab"=>"Matlab", "mathematica"=>"Mathematica",
+                "python"=>"Python", "octave"=>"Octave", "r"=>"R", "rust"=>"Rust", "go"=>"Go");
+    benchmarks[!,:language] = [dict[lang] for lang in benchmarks[!,:language]]
+
+    # Normalize benchmark times by C times
+    ctime = benchmarks[benchmarks[!,:language].== "C", :]
```

Comment:
I'd recommend symmetric spacing around the equality:
```suggestion
    ctime = benchmarks[benchmarks[!,:language] .== "C", :]
```